charts/intel-qat-resource-driver/values.yaml

```diff
@@ -19,14 +19,22 @@ serviceAccount:
 
 kubeletPlugin:
   podAnnotations: {}
-  tolerations:
-    - key: node-role.kubernetes.io/master
-      operator: Exists
-      effect: NoSchedule
-    - key: node-role.kubernetes.io/control-plane
-      operator: Exists
-      effect: NoSchedule
   nodeSelector:
-    {}
+    feature.node.kubernetes.io/qat: "true"
     #node-role.kubernetes.io/control-plane: ""
+  tolerations:
+    # - key: node-role.kubernetes.io/master
+    #   operator: Exists
+    #   effect: NoSchedule
+    # - key: node-role.kubernetes.io/control-plane
+    #   operator: Exists
+    #   effect: NoSchedule
+    - key: "node.kubernetes.io/qat"
+      operator: "Exists"
+      effect: "NoSchedule"
   affinity: {}
+
+nfd:
+  enabled: false # change to true if NFD is not installed in cluster
+  nameOverride: intel-qat-nfd
+  enableNodeFeatureApi: true
```

Comment:
**enableNodeFeatureApi** is used instead of **nodeFeatureRules.enabled** simplifying the management of the chart. nameOverride is used to set custom name for nfd components themselves, replacing default values

Reply:
Ok, I found `nameOverride` use from `templates/_helpers.tpl` (= already in repo), but I still do not see anything using `enableNodeFeatureApi`, neither in this PR, or already in repo.  Where it's used?

Reply:
@eero-t , this is "nfd" section, nfd is an alias for https://kubernetes-sigs.github.io/node-feature-discovery/v0.16/deployment/helm.html?highlight=enableNodeFeatureApi#general-parameters , these are values for the dependency chart with one exception: line 14 in `chart.yaml` tells to install this dependency when `nfd.enabled` is true - `enabled` param is for this current chart.

Reply:
Comment in the indicated docs state:
> DEPRECATED: will be removed in NFD v0.17, use `featureGates.NodeFeatureAPI` instead.

Looking at the older NFD docs,  `enableNodeFeatureApi` was referring to gRPC API deprecated already in NFD v0.14, but the `featureGates` options were added only in v0.16.

Reply:
```suggestion
  nameOverride: intel-qat-nfd
  # TODO: this deprecated NFD option will be replaced with "featureGates.NodeFeatureAPI" in NFD v0.17:
  # https://kubernetes-sigs.github.io/node-feature-discovery/v0.16/deployment/helm.html#general-parameters
  enableNodeFeatureApi: true
```